src/components/SQForm/SQForm.tsx

```diff
@@ -37,6 +37,10 @@ export type SQFormProps<Values extends FormikValues> = {
    * https://jaredpalmer.com/formik/docs/guides/validation#validationschema
    * */
   validationSchema?: AnyObjectSchema;
+  // Set if validation should happen on blur or not.  By default is set to true.
+  validateOnBlur?: boolean;
+  // Set if validation should happen on change.  By defautl is set to true.
```

Comment:
nit: typo default

Also, could we mention that it's set to `true` by default in Formik and not in our code so it doesn't get confusing if someone is reading this documentation when we're clearly not setting it to default true on our own.

Reply:
great point and good catch on the typo!  will make this change now.

Reply:
pushed 👍